books/free-programming-books-langs.md

```diff
@@ -447,6 +447,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [C Internals](https://www.avabodh.com/cin/cin.html) - Rajeev Kumar (HTML)
 * [C Notes for Professionals](https://goalkicker.com/CBook) - Compiled from StackOverflow Documentation (PDF)
 * [C Programming](https://en.wikibooks.org/wiki/Programming%3AC) - Wikibooks
+* [C Programming Absolute Beginner's Guide (3rd Edition)](https://usermanual.wiki/Pdf/CProgrammingAbsoluteBeginnersGuide3rdEditio.424140197.pdf) - Greg Perry and Dean Miller (PDF)
```

Comment:
This is not an authorized distribution. Refer to docs/CONTRIBUTING.md for guidelines.